common/notification/actions.ts

```diff
@@ -638,15 +638,11 @@ const _notificationActions = {
     },
     joined_match_meeting: {
         description: 'User joined a match meeting',
-        sampleContext: {
```

Comment:
I think I would still collect further data in context such as the `matchId / subcourseId  /matchpartner` etc.

Reply:
and we do in the mutations file. As I see it, this is just a sample and not responsible for fetching all necessary data, as this is done by the mutation. Am I wrong?

Reply:
I talked to our beloved boss and he explained to me that the sample context is just for visualization and is never used. I adjusted my code accordingly.

<img src="https://media4.giphy.com/media/tIeCLkB8geYtW/giphy.gif"/>

Reply:
"and he explained to me that the sample context is just for visualization and is never used" - This is not correct. The sample context is used to validate Notifications created for a certain action to ensure all templating done is correct. Also it is used to infer the type of `SpecificNotificationContext`. So please maintain this.